atests/testcase.robot

```diff
@@ -323,3 +323,19 @@ Post Request With Large Truncated Body
     ${html}=  Get File  ${CURDIR}${/}index.html
     ${resp}=  Post Request  ${test_session}  /anything  data=${html}
     Status Should be  200  ${resp}
+
+Last response
+    [Tags]  last_response
+    Create Session  github  https://api.github.com   verify=${CURDIR}${/}cacert.pem
+    Get Request  github  /users/bulkan
+    Should Be Equal As Strings  ${last_response.status_code}  200
+    Dictionary Should Contain Value  ${last_response.json()}  Bulkan Evcimen
+
+New Last response
+    [Tags]  last_response
+    [Documentation]   Given a new request has been done
+    ...               Then ${last_response} should be equal to latest request's response
+    Create Session  jsonplaceholder  https://jsonplaceholder.typicode.com
```

Comment:
better to use the ${GLOBAL_SESSION} generated in setup to avoid real external communication during the build